text/0000-release-process.md

```diff
@@ -0,0 +1,319 @@
+
+More commonly Feature-Complete User Acceptance is used in practice because any additional changes could have unexpected
+side-effects invalidating prior completed UATs. In our case, Feature-Complete UAT fit perfectly during the
+[feature complete](#feature-complete) phase.
+
+#### Feature Complete
+
+Feature complete for a particular release is a period of time when changes are analyzed and vetted before being added to
+a release.
+
+The following take place:
+
+1. At the beginning, a release branch is created (`release/<version>`).
+2. At the beginning, a [change control board](#change-control-board) is organized on a daily basis to review additional
+requested changes.
+3. During, only changes approved by the CCB are merged into the release branch.
+
+#### Change Control Board
+
+A change control board would consist of the maintainers of the components's sub-team. They would organize
+changes that might be suitable for [release branches](#release-branches) during feature complete. The communication of
+the board could take place in Slack, such as in `#release-planning`, or through meetings organized by the
+[release manager](#release-manager).
+
+The board members typically decide on whether a change should be included in the release based on the following criteria:
+
+- Impact
+- Effort
+- Risk
+
+#### Release branches
+
+Release branches are protected branches that will be created at the beginning of feature complete. These branches are
+protected, similar to `master`, if not more so depending on the [change control board](#change-control-board) members. 
+
+The release branch will be tagged and merged into `master` upon release. At this point in time, there is no intention to
+maintain the release branch after it has been merged in. For more details, refer to [release finalization](#release-finalization)
+
+```text
+                 master
+
+                   o
+                   |
+                   |    release/1.2.3
+                   |
+                   o ---------+     <--+ merge release branch into master
+                   |          |
+                   |          o commit: fix/critical-or-minor (tag:v1.2.3)
+                   |          |
+                   |          |
+commit: feature/Y  o          |
+                   |          |
+                   |          |
+                   |          |
+                   |          |
+                   +----------+     <--+ start feature complete
+                   |
+                   |
+                   |
+                   |
+commit: feature/X  o
+                   |
+                   |
+                   +
+```
+
+#### Migration Guide
+
+A migration guide is a document which details breaking changes and provides actions that may be taken to migrate from
+prior versions.
+
+### Release Manager
+
+A release manager is a role assigned to an individual selected to own the release process. A release manager should be
```

Comment:
Is the release manager assigned per release or is this a more long standing role?